json-schemas/permissions/scopes.json

```diff
@@ -3,6 +3,17 @@
   "$id": "https://identity.foundation/dwn/json-schemas/permissions/scopes.json",
   "type": "object",
   "$defs": {
+    "events-query-scope": {
+      "type": "object",
+      "properties": {
```

Comment:
Sanity: missing `protocol`?

Reply:
Good catch, fixed this and slightly improved the others that were lacking as well.